### DISCUSSION TITLE
Sobre github series

### DISCUSSION THREAD

**Discussion by cmjosueh** [Launch-X-Latam/MisionFrontEnd#346] [Q&A]
Hola que tal, oigan el dia de ayer estuve en el workshop de github series, pero al final en la sección de dudas ya no me pude quedar. lo que pasa es que me interesa si sí van a compartir la grabación ¿alguien sabe que onda al respecto?

**Comment by JoseEduardoVelazquezCN** [comment]
Mencionaron que mandarían un correo con la grabación, aunque aún no lo recibo 🥲

**Comment by doguedogue** [comment]
No lo han enviado, que duda tienes?

**Comment by doguedogue** [comment]
aqui está el video [link](https://github.zoom.us/rec/play/y3juY7M2kBkBDQlNtjvCY8LRHuFuIYk4y-cyGvK0n0lf3lOnvP2ROAaqeW3Jo7d64vAadbOa6BECK_i0.a9dHJYRtidEvyFq5?continueMode=true&_x_zm_rtaid=Oz3LwecVQseVMv6rGQGApA.1667838753184.a71d7c918794be0251d6ded1aa10f2f9&_x_zm_rhtaid=614)
